plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergSystemTables.java

```diff
@@ -332,7 +332,7 @@ public void testSnapshotsTable()
     @Test
     void testAllManifests()
     {
-        try (TestTable table = newTrinoTable("test_all_manifests", "AS SELECT 1 x")) {
+        try (TestTable table = newTrinoTable("test_all_manifests", "(\"x\") AS VALUES 1, 2")) {
```

Comment:
Remove redundant double quotations for `x`.

Reply:
Both done